pywbemtools/pywbemcli/_pywbem_server.py

```diff
@@ -164,23 +164,24 @@ def __init__(self, server=None, default_namespace=DEFAULT_NAMESPACE,
 
         # WBEMServer object when connected; None when disconnected.
         self._wbem_server = None
+        self._conn = None
```

Comment:
`self._ conn` is redundant to `self._wbem_server.conn`.

I cannot see a reason to have this redundant attribute and suggest to not introduce it again (after PR #900 just eliminated it).

Reply:
Agreed. Done. Conn property exists but just uses wbem_server as the flag